pkg/api/v1alpha1/cluster.go

```diff
@@ -698,6 +693,28 @@ func validateMirrorConfig(clusterConfig *Cluster) error {
 	if clusterConfig.Spec.RegistryMirrorConfiguration.InsecureSkipVerify && clusterConfig.Spec.DatacenterRef.Kind != SnowDatacenterKind {
 		return errors.New("insecureSkipVerify is only supported for snow provider")
 	}
+
+	mirrorCount := 0
+	ociNamespaces := clusterConfig.Spec.RegistryMirrorConfiguration.OCINamespaces
+	for _, ociNamespace := range ociNamespaces {
+		if ociNamespace.Registry == "" {
+			return errors.New("registry can't be set to empty in OCINamespaces")
+		}
+		if re.MatchString(ociNamespace.Registry) {
+			mirrorCount++
+			// More than one curated package mirrors would confuse package controller where to pull packages
```

Comment:
```suggestion
			// More than one mirror for curated package would introduce ambiguity in the package controller
			// when deciding where to pull packages from, since it only allows for one source registry
```